lib/domain/action.js

```diff
@@ -19,6 +26,7 @@ export default class Action {
     this.dueDate = IsoDate.parse(dueDate);
     this._isCompleted = isCompleted;
     this.completedDate = IsoDateTime.parse(completedDate);
+    this.completedBy = completedBy;
```

Comment:
In usePlan it checks whether it's completed or not and passes relevant value

`{ isCompleted, completedBy: isCompleted ? getUsername(token) : null },`

Reply:
Ok cool, I think it might be worth moving this to the API-side still though - if the API is called directly at some point in the future it would be nice if this was consistently applied. Whereas by having it done on the client-side it will only apply for requests that come through the shared plan UI itself.